advanced_html_css/animation/transitions.md

```diff
@@ -24,7 +24,7 @@ CSS transitions let you animate a change from an element's initial state to an e
 
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-When your mouse cursor is away from the button, the button is in the initial state. When you hover over it you introduce the end state, the hover state, causing the transition of the color smoothly fading from white to black to occur.
+When your mouse cursor is away from the button, the button is in the initial state. When you hover over it you introduce the end state, the hover state, causing the transition of background-color smoothly from white to black.
```

Comment:
Nit for the whole sentence while we're at it. What do you think?
```suggestion
When your mouse cursor is away from the button, the button is in the initial state. When you hover over it, you introduce the end state - the hover state - causing the background color to transition smoothly from white to black.
```

Reply:
> Nit for the whole sentence while we're at it. What do you think?

looks good, I'd merge

Reply:
Then you can make this change and commit it, either locally then pushing, or via the "commit suggestion" button in my review comment. Once commited, I can merge to main.

Reply:
> Then you can make this change and commit it, either locally then pushing, or via the "commit suggestion" button in my review comment. Once commited, I can merge to main.

I'll get to creating a new PR since I can't commit to this one. As a side not, do you think it should be 'end state' or 'final state'?

Reply:
You do not need to make a new PR. In this case, the simplest thing to do is hit the "commit suggestion" button under my suggestion above.

Reply:
> Then you can make this change and commit it, either locally then pushing, or via the "commit suggestion" button in my review comment. Once commited, I can merge to main.

there might be a misunderstanding, I didn't create this PR

Reply:
Oh I see, I mistook you for the PR author. In that case, you should not create a duplicate PR. My review comment was aimed at the PR author so I'm awaiting their response/action.